bokeh/application/application.py

```diff
@@ -23,7 +23,7 @@ def create_document(self):
             h.modify_document(doc)
             if h.failed:
                 # TODO (havocp) this could be formatted a lot more nicely
```

Comment:
if this TODO is still true, I guess it's now less obvious what "more nicely" means, so maybe the comment should go or be more specific.